packages/notifi-react-card/lib/components/intercom/SendMessageSection.tsx

```diff
@@ -31,6 +42,7 @@ export const SendMessageSection: React.FC<SendMessageSectionProps> = ({
       )}
     >
       <textarea
+        onKeyDown={(e) => handleKeypressDown(e)}
```

Comment:
small, but in my opinion important, detail: we should use `keyup` rather than keydown.

Reply:
Cool! Had to remind myself of `keyUp` vs. `keyDown`. I agree, `onKeyUp` would be better since it would happen after release

Reply:
learned😊